source/tunneling/TcpForward.cpp

```diff
@@ -55,14 +57,6 @@ namespace Aws
                     return 0;
                 }
 
-                int TcpForward::Close()
-                {
-                    FlushSendBuffer();
```

Comment:
Is there a need to call FlushSendBuffer() in the new destructor definition? If no, then resolve this comment.